Resources/Locale/en-US/store/uplink-catalog.ftl

```diff
@@ -28,13 +28,13 @@ uplink-gloves-north-star-desc = A pair of gloves that reduce your punching coold
 
 # Explosives
 uplink-explosive-grenade-name = Explosive Grenade
-uplink-explosive-grenade-desc = Grenade that creates a small but devastating explosion.
+uplink-explosive-grenade-desc = A simplistic grenade with a ten-second fuse that is geared towards injuring personnel. Causes minimal hull damage.
 
 uplink-flash-grenade-name = Flashbang
-uplink-flash-grenade-desc = Eeeeeeeeeeeeeeeeeeeeee
+uplink-flash-grenade-desc = A standard-issue flashbang, capable of blinding and slowing down anyone without proper protection. This, of course, includes you; make sure you're properly equipped before using it.
```

Comment:
```suggestion
uplink-flash-grenade-desc = Eeeeeeeeeeeeeeeeeeeeee
```

You don't have to remove the memey explanation. Some of it is part of the spirit of the game.

Reply:
It's just a copy of the item description, like most other uplink items. The comedy is already in there so I'm not sure if it's worth having both as it just ends up being weirdly non-descriptive all-around.

Reply:
Should be fine then.